influence_benchmark/config/experiment_configs/EI_nudging_therapist.yaml

```diff
@@ -3,11 +3,11 @@ run_name: "EI_nudging_therapist"
 log_to_wandb: true
 
 # Specify settings for generating trajectories
-env_class: "nudging_therapist"
+env_class: "nudging-therapist"
```

Comment:
Very minor but `nudging-therapist` is the only env class with `-` instead of `_`

Reply:
I guess i felt nudging-therapist is a different environment from therapist whereas gul_therapist etc are just different initial states